tg_bot.py

```diff
@@ -45,13 +47,17 @@ def display_tweet(self, context):
                 u = self.twitter_api.get_status(id=Id['Id'])
                 username = u.author.screen_name
             except tweepy.TweepError as e:
+                if self.level == logging.CRITICAL:
+                        with open(self.dirpath + "error.log", "a") as fp:
+                            fp.write('ERROR:' + time.strftime("%m/%d/%Y %I:%M:%S %p ") + "\t|Tweepy failed to get the status of the user from the " +
+                                 str(Id['Id']) + " because of " + e.reason + "\n")
                 self.nw_logger.logger('\t|Tweepy failed to get the status of the user from the ' +
                                         str(Id['Id']) + ' because of ' + e.reason + '\n\n', 'error', 'Error')
                 pass
             url = 'https://twitter.com/' + \
                 username + '/status/' + str(Id['Id'])
                 
-            if job.context in config.tg_admin_id:
+            if job.context in config.tg_admin_id and self.auto_retweet == False or self.auto_retweet == None:
```

Comment:
if (job.context in config.tg_admin_id) and (self.auto_retweet == False or self.auto_retweet == None):

Reply:
I added the brackets